src/main/java/website/frontrow/util/CollisionHandler.java

```diff
@@ -246,9 +246,50 @@ public Collision findNextPosition(Mover mover)
 			mover.onWallCollision();
 		}
 
-		return new Collision(new Point(
-					Math.round(collision.getPoint().getX() * ONE_DEV_PRECISION) * PRECISION,
-					Math.round(collision.getPoint().getY() * ONE_DEV_PRECISION) * PRECISION
-			), collision.isCollided());
+        double newXLocation
+                = Math.round(collision.getPoint().getX() * ONE_DEV_PRECISION) * PRECISION;
```

Comment:
No not by an significant amount. Since every grid cell is 1 generic unit wide and every unit is equal to 32 pixels, the player will not notice the error.